internal/repo/pg/url.go

```diff
@@ -0,0 +1,163 @@
+package pg
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azzonya/go-shortener/internal/entities"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"log"
+)
+
+type St struct {
+	db *pgxpool.Pool
+}
+
+func New(db *pgxpool.Pool) *St {
+	s := &St{
+		db: db,
+	}
+
+	var err error
+	exist := s.TableExist()
+
+	if !exist {
```

Comment:
```suggestion
	if !s.TableExist() {
```
Но вообще можно не проверять а сделать через `CREATE TABLE IF NOT EXIST`